src/darwin/CHIPTool/CHIPTool/View Controllers/QRCode/QRCodeViewController.m

```diff
@@ -672,18 +672,24 @@ - (void)commissionWithSSID:(NSString *)ssid password:(NSString *)password
     }
 }
 
-- (void)onCommissioningComplete:(NSError * _Nullable)error
+// MARK: MTRDeviceControllerDelegate
+- (void)controller:(MTRDeviceController *)controller commissioningComplete:(NSError *)error nodeID:(NSNumber *)nodeID
```

Comment:
Again, this is missing the _Nullable annotations.

Reply:
I added.